content/quests/events/demevent_lobbying.lua

```diff
@@ -19,48 +19,66 @@ local QDEF = QuestDef.Define
+                * You notice the jingling of shills before you notice the person holding them.
+                * This person likely has some deep pockets, and deep pockets mean-
+                merchant:
+                    !right
+                    !point
+                    I'm going to stop you right there before you try to rob me.
                 player:
                     !left
+                    What, do you really think I, a professional politician, would resort to petty theft?
                 merchant:
-                    !right
-                    Yo, I heard you are running a campaign.
-                    So I would like to provide some funds for you.
+                    !crossed
+                    ...
                 player:
-                    What's the catch?
+                    ...
+                    !hips
+                    Alright, what's the deal?
                 merchant:
-                    The catch is that you must support my position.
+                    The deal you presume I want is something simple.
+                    If you publicly support {1#pol_stance} for me, i'll provide you some campaign funds.
             ]],
             OPT_ACCEPT = "Accept",
             DIALOG_ACCEPT = [[
                 player:
-                    [p] I wouldn't turn away free money.
+                    !take
+                    You give me money and all I have to do is give you my word. Seems like a great deal!
                 merchant:
-                    Excellent! That's the sort of stuff I like to see.
+                    !wink
+                    Your word is quite valuable.
             ]],
             OPT_DECLINE = "Decline",
             DIALOG_DECLINE = [[
                 player:
-                    [p] Nah I don't think I will take it.
+                    I've still got some dignity. I'll pass on this.
                 merchant:
-                    I thought you are shrewd.
+                    !shrug
+                    Well, I guess you just want to let this money fall into your opponent's hands.
+                    !exit
+                * {merchant} walks away, jingling the large sum of shills in {merchant.hisher} pockets louder just to spite you.
             ]],
             OPT_ASK_FOR_MORE = "Ask for more money",
             DIALOG_ASK_FOR_MORE = [[
                 player:
-                    [p] I am frankly insulted to think that you can bribe me with this little money.
-                    How much are you willing to offer, hmm?
+                    Listen, i'm flattered, but you understand the impact this will have on my voting base, correct?
+                    I'll need a bit more to help keep my campaign afloat.
             ]],
             DIALOG_ASK_FOR_MORE_SUCCESS = [[
+                player:
+                    !eureka
+                    If I end up losing, this'll be a wasted investment. If I have enough money, i'll be able to stay in the race longer.
```

Comment:
Also the "I" here.

Reply:
Also not sure the contraction "this'll" actually exist. The spell checker says it doesn't, so you probably want to take a look at it.